techs/zetapack/factions/romans/upgrades/advanced_architecture/advanced_architecture.xml

```diff
@@ -4,7 +4,10 @@
 	<image path="images/advanced_architecture.bmp"/>
 	<image-cancel path="images/tech_upgrade_cancel.bmp"/>
 	<time value="100"/>
-	<unit-requirements/>
+	<unit-requirements>
+		<unit name="temple"/>
+		<unit name="blacksmith_shop"/>
+	</unit-requirements>
 	<upgrade-requirements/>
 	<resource-requirements>
 		<resource name="wood" amount="100"/>
```

Comment:
changes to rome all look reasonable. But you mentioned something about using spearmen to spam? You think this might prevent that? Should we look at giving a slight increase to spearmen or changing some other element to prevent spamming with spearmen?

Also, if you think rome has become too strong, we may want to evaluate if attack damage from catapults should be reduced (since their range has been increased).

Reply:
It prevents early spam, not later spam